iogt_content_migration/management/commands/load_v1_users.py

```diff
@@ -0,0 +1,387 @@
+
+    def db_connect(self, options):
+        connection_string = self.create_connection_string(options)
+        self.stdout.write(f'DB connection string created, string={connection_string}')
+        self.v1_conn = psycopg2.connect(connection_string)
+        self.stdout.write('Connected to v1 DB')
+
+    def __del__(self):
+        try:
+            self.v1_conn.close()
+            self.stdout.write('Closed connection to v1 DB')
+        except AttributeError:
+            pass
+
+    def get_query_results_count(self, sql):
+        sql = f'select count(*) from ({sql}) as count_table'
+        cur = self.db_query(sql).fetchone()
+        return cur['count']
+
+    def with_progress(self, sql, iterable, title):
+        self.stdout.write(title)
+        return tqdm(iterable, total=self.get_query_results_count(sql))
+
+    def request_registration_survey_mandatory_groups(self):
+        sql = f'select * from auth_group'
+        cur = self.db_query(sql)
+
+        self.stdout.write('==============================')
+        self.stdout.write('User Groups in v1:')
+        [self.stdout.write(f'{group["id"]} - {group["name"]}') for group in cur]
+
+        self.stdout.write('\n Please mention the groups for which you want to mark the registration survey'
+                          'mandatory?')
+        group_ids = raw_input('(Use comma separated ids, leave blank to make it optional)')
+
+        if group_ids:
+            group_ids = group_ids.split(',')
+            self.stdout.write(f'\n The script will mark registration survey mandatory for all'
+                              f' users of group_ids: {group_ids}')
+        else:
+            self.stdout.write(f'\n The script will mark registration survey optional for all users')
+            group_ids = []
+
+        sleep(5)
+        return group_ids
+
+    def create_connection_string(self, options):
+        host = options.get('host', '0.0.0.0')
+        port = options.get('port', '5432')
+        name = options.get('name', 'postgres')
+        user = options.get('user', 'postgres')
+        password = options.get('password', '')
+        return f"host={host} port={port} dbname={name} user={user} password={password}"
+
+    def db_query(self, q):
+        cur = self.v1_conn.cursor(cursor_factory=psycopg2.extras.RealDictCursor)
+        cur.execute(q)
+        return cur
+
+    def migrate(self):
+        self.populate_content_type_map()
+        self.populate_articles_map()
+        self.populate_surveys_map()
+
+        self.migrate_user_groups()
+        self.migrate_user_accounts()
+        self.mark_user_registration_survey_required()
+        self.populate_users_map()
+
+        self.migrate_user_comments()
+        self.migrate_user_submissions()
+        self.migrate_user_poll_submissions()
+
+    def get_mapping_from_title(self, klass, title):
+        return klass.objects.get(title=title)
+
+    def get_mapping_from_path(self, klass, path, title):
```

Comment:
Is there no better way to map surveys/polls than title/path?
Can we not create a table as part of load_v1_db that creates a map from v1 survey/poll IDs to v2 survey/poll IDs?

What I'm envisioning is country offices moving content around or possibly even renaming content as part of the curation of the v1->v2 migrated content, causing new submissions to get mismatched.